docs/excel/excel-data-types-concepts.md

```diff
@@ -65,21 +70,27 @@ const myEntity = {
             type: Excel.CellValueType.string,
             basicValue: "I love llamas."
```

Comment:
Getting some serious SimCity flashbacks here ;-)

Reply:
Haha, maybe that's why the engineers included llamas in so many code samples!